tests/integration/targets/ndo_route_map_policy_route_control/tasks/main.yml

```diff
@@ -0,0 +1,380 @@
+          - nm_add_route_map_policy.proposed.templateId is defined
+          - nm_add_route_map_policy.proposed.templateName == "ansible_tenant_template"
+          - nm_add_route_map_policy_again is not changed
+          - nm_add_route_map_policy_again.current.description == ""
+          - nm_add_route_map_policy_again.current.name == "route_map_policy_1"
+          - nm_add_route_map_policy_again.current.routeMapType == "global"
+          - nm_add_route_map_policy_again.current.templateId is defined
+          - nm_add_route_map_policy_again.current.templateName == "ansible_tenant_template"
+          - nm_add_route_map_policy_again.current.uuid is defined
+          - nm_add_route_map_policy_again.previous.description == ""
+          - nm_add_route_map_policy_again.previous.name == "route_map_policy_1"
+          - nm_add_route_map_policy_again.previous.routeMapType == "global"
+          - nm_add_route_map_policy_again.previous.templateId is defined
+          - nm_add_route_map_policy_again.previous.templateName == "ansible_tenant_template"
+          - nm_add_route_map_policy_again.previous.uuid is defined
+          - add_route_map_policy_2 is changed
+          - add_route_map_policy_2.current.description == "test"
+          - add_route_map_policy_2.current.name == "route_map_policy_2"
+          - add_route_map_policy_2.current.routeMapType == "combinable"
+          - add_route_map_policy_2.current.templateId is defined
+          - add_route_map_policy_2.current.templateName == "ansible_tenant_template"
+          - add_route_map_policy_2.current.uuid is defined
+          - add_route_map_policy_2.previous == {}
+
+    # UPDATE
+    - name: Update Route Map Policy for Route Control 1 (check_mode)
+      cisco.mso.ndo_route_map_policy_route_control: &cm_update_route_map_policy
+        <<: *mso_info
+        template: ansible_tenant_template
+        uuid: "{{ nm_add_route_map_policy.current.uuid }}"
+        name: route_map_policy_1_updated
+        description: updated
+        match_type: prefix_and_routing_policy
+        state: present
+        output_level: debug
+      check_mode: true
+      register: cm_update_route_map_policy
+
+    - name: Update Route Map Policy for Route Control 1
+      cisco.mso.ndo_route_map_policy_route_control:
+        <<: *cm_update_route_map_policy
+      register: nm_update_route_map_policy
+
+    - name: Update Route Map Policy for Route Control 1 again
+      cisco.mso.ndo_route_map_policy_route_control:
+        <<: *cm_update_route_map_policy
+      register: nm_update_route_map_policy_again
+
+    - name: Update Route Map Policy for Route Control 1 using name
+      cisco.mso.ndo_route_map_policy_route_control:
+        <<: *mso_info
+        template: ansible_tenant_template
+        name: route_map_policy_1_updated
+        description: updated_again
+        match_type: routing_policy_only
+        state: present
+      register: update_route_map_policy_with_name
+
```

Comment:
Can you add a test case that verifies all the removable properties gets removed from object?